package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "tapestry-lite",
-  "version": "1.1.1",
+  "version": "1.1.2",
```

Comment:
1.2.0 for a new feature. 1.1.2 would imply 1.1.1 had a proxy route I feel.

Reply:
Ah yeah, that was my intention, this is an old `npm version` update that I hadn't pushed yet.